BLSQ/iaso
# IA-1623: add a new workflow to launch cypress tests

Explain what problem this PR is resolving

## Self proof reading checklist

## Changes

New workflow based on the one we use for cypress manual testing (with comment on a PR). 
I added the event trigger to launch the workflow from github actions with this event : `workflow_dispatch
`
I added a condition to mark the PR and comment only if you triggered the `issue_comment` event by adding a comment with the body : @cypress 

## How to test

Go to [Actions](https://github.com/BLSQ/iaso/actions). In the list of workflows, you should see a new "Cypress Manual Testing" workflow you can click on it.
